modules/3d-tiles/package.json

```diff
@@ -38,6 +38,7 @@
     "@loaders.gl/gltf": "2.0.0-beta.8",
     "@loaders.gl/loader-utils": "2.0.0-beta.8",
     "@loaders.gl/math": "2.0.0-beta.8",
+    "@deck.gl/core": "^8.0.0-beta.2",
```

Comment:
This is the offending line...

Reply:
Thanks for catching this. deck.gl should be a devDependency, since it’s only imported by a test. For test purposes, @math.gl/web-mercator should be sufficient as well.

Reply:
But `Viewport` in `@math.gl/web-mercator` does not have camera related fields, like `cameraPosition` etc.